test/uk/gov/hmrc/plasticpackagingtax/registration/views/group/OrganisationDetailsTypeViewSpec.scala

```diff
@@ -66,6 +69,13 @@ class OrganisationDetailsTypeViewSpec extends UnitViewSpec with Matchers {
       view.select("title").text() must include(messages("organisationDetails.other.group.title"))
     }
 
+    "display title for next group member" in {
+      val view: Document = createView(OrganisationType.form(), false)
```

Comment:
Maybe name the `isFirstMember` param to aid readability?

Reply:
For me intellij shows the parameter name, so i did not name. I have done it now.